pkg/terraform/util.go

```diff
@@ -87,3 +88,40 @@ func CreateTerraformBackendFile(assetDir, data string) error {
 
 	return nil
 }
+
+// PrepareTerraformRootDir creates a directory named path including all
+// required parents.
+// An error is returned if the directory already exists.
+func PrepareTerraformRootDir(path string) error {
+	pathExists, err := util.PathExists(path)
+	if err != nil {
+		return errors.Wrapf(err, "failed to stat path %q: %v", path, err)
```

Comment:
This should not introduce use of `errors.Wrapf` ideally, if it is removed in the next commit. This pollutes git history.